### DISCUSSION TITLE
Calibre 6.3 and DeDRM v10.0.3 -> removal Problem

### DISCUSSION THREAD

**Discussion by synthesechess** [apprenticeharper/DeDRM_tools#2123] [General]
Hi,
I'm trying to remove/ convert the DRM from a pdf. It's not working but I got this log:

Any ideas for a solution?

Thank you so much!

Chris

**Comment by ElleKayEm** [comment]
Since you're using DeDRM 10.0.3, you might want to ask over at noDRM's repo instead of here at Apprentice Harper's which ended with 7.2.1.

**Reply by synthesechess** [reply]
Sorry, but do not understand. Can you explain?
Tank you so much.

**Reply by ElleKayEm** [reply]
You're posting on Apprentice Harper's GitHub repository which is no longer maintained.  Last release was DeDRM 7.2.1.  The version you're using is 10.0.3 from noDRM:  https://github.com/noDRM/DeDRM_tools

**Reply by synthesechess** [reply]
Understand. Thx!
